edit-and-sync-features/README.md

```diff
@@ -0,0 +1,47 @@
+# Edit and sync features
+
+This sample demonstrates how to synchronize offline edits with a feature service.
+
+## How to use the sample
+
+![](editandsyncfeatures.png)
+
+- Pan and zoom into the desired area, making sure the area you want to take offline is within the current extent of the `MapView`. 
+- Tap on the Generate Geodatabase button. This will call `generateGeodatabase()`, which will return a `GenerateGeodatabaseJob`. 
+- Once the job completes successfully, a `GeodatabaseFeatureTable` and a `FeatureLayer` are created from the resulting `Geodatabase`. The `FeatureLayer` is then added to the `ArcGISMap`. 
+- Once the `FeatureLayer` generated from the local `Geodatabase` is displayed, a `Feature` can be selected by tapping on it. The selected `Feature` can be moved to a new location by tapping anywhere on the map. 
+- Once a successful edit has been made to the `FeatureLayer`, the Sync Geodatabase button is enabled. This button synchronizes local edits made to the local `GeoDatabase` with the remote feature service using `syncGeodatabase()` which generates `SyncGeodatbaseParameters` and passes them to a `SyncGeodatabaseJob`.
+- Once the job successfully completes, the local edits are synchronized with the feature service.
+
+## Features
+
+* MapView
+* Map
+* Basemap
+* ArcGISTiledLayer
+* TileCache
```

Comment:
Since this is about syncing features let's drop from here up as they are not focused features of the sample